templates/master/00-master/on-prem/files/keepalived-keepalived.yaml

```diff
@@ -69,6 +69,63 @@ contents:
         fall 2
     }
 
+    # When using dual-stack with OpenStack, both IPv4 and IPv6 share the same Neutron Port and this
+    # makes OVN thinks that both addresses are associated to the same Node, but that might not always be
+    # true. To change that, let's make sure the API VIPs stays together through state changes, the same
+    # goes for Ingress VIPs.
```

Comment:
Proposed rewrite:

```
When using dual-stack with OpenStack, both IPv4 and IPv6 share the same Neutron Port, causing OVN to assume both addresses belong to the same node, which may not always be the case. To address this, we ensure that the API VIPs remain grouped through state changes, along with the Ingress VIPs.
```

Reply:
updated. I just changed the end of the sentence a bit.